### PR TITLE
wicked: Rename file to adhere to the test plan

### PR DESCRIPTION
**Pull request by schlad**
Add README document to describe overall test plan for wicked
testing.

### CODE REVIEW

File: tests/wicked/README [hunk 1]
```diff
@@ -0,0 +1,16 @@
+Wicked testing in openQA follows earlier test approach done in wicked-testsuit.
```
Comment (asmorodskyi) [inline tests/wicked/README]: fix typo `wicked-testsuitE`

Comment (schlad) [reply tests/wicked/README]: :) forgot about this

Comment (schlad) [reply tests/wicked/README]: changed and v2 sent

**Comment by okurz** [comment]
+1 . please remember to update the openQA test database settings on o3 and osd after merging

**Pull request merged**